zello-sdk/src/main/java/com/zello/sdk/Audio.java

```diff
@@ -179,11 +166,6 @@ private void updateAudioState(@Nullable Intent intent) {
 		String mode = Util.emptyIfNull(intent.getStringExtra(Constants.EXTRA_MODE));
 		if (mode.startsWith(Constants.EXTRA_WA)) {
 			_mode = AudioMode.WEARABLE;
-			try {
-				_wearable = Integer.parseInt(mode.substring(Constants.EXTRA_WA.length()));
-			} catch (NumberFormatException ignore) {
-				_wearable = -1;
-			}
```

Comment:
Not only wearables are no longer supported, but also the currently selected wearable index was never utilized.